### PR TITLE
[ui] Remove old LogsScrollingTable

### PR DESCRIPTION
**Pull request by hellendag**
## Summary & Motivation

Replace last callsite of `LogsScrollingTable` with the new version, remove the old component. Followup will rename the new one.

## How I Tested These Changes

Perform a materialization, view logs via the sidebar in the global asset graph. Verify that logs table behaves (scrolls, resizes, pins scrolling) correctly during and after the run.

### CODE REVIEW

**Comment by hellendag** [comment]
* **#22956** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/22956?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @hellendag and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by salazarm — APPROVED** [review]
This is all existing code right?

**Comment by hellendag** [comment]
Yeah exactly. I'll make some comments inline for posterity.

File: js_modules/dagster-ui/packages/ui-core/src/runs/filterLogs.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import {LogFilter, LogsProviderLogs} from './LogsProvider';
+import {eventTypeToDisplayType} from './getRunFilterProviders';
+import {logNodeLevel} from './logNodeLevel';
+
+export function filterLogs(logs: LogsProviderLogs, filter: LogFilter, filterStepKeys: string[]) {
```
Comment (hellendag) [inline js_modules/dagster-ui/packages/ui-core/src/runs/filterLogs.tsx:5]: This is moved from the now-deleted `LogsScrollingTable`.

File: js_modules/dagster-ui/packages/ui-core/src/runs/LogsScrollingTableNew.tsx [hunk 2]
```diff
@@ -110,3 +126,11 @@ export const LogsScrollingTableNew = (props: ILogsScrollingTableProps) => {
     </ColumnWidthsProvider>
   );
 };
+
+export const ListEmptyState = styled.div`
```
Comment (hellendag) [inline js_modules/dagster-ui/packages/ui-core/src/runs/LogsScrollingTableNew.tsx:130]: This is moved from the now-deleted `LogsScrollingTable`.

File: js_modules/dagster-ui/packages/ui-core/src/runs/LogsScrollingTableNew.tsx [hunk 3]
```diff
@@ -1,15 +1,31 @@
...
-import {ILogsScrollingTableProps, ListEmptyState, filterLogs} from './LogsScrollingTable';
 import {ColumnWidthsProvider, Headers} from './LogsScrollingTableHeader';
+import {IRunMetadataDict} from './RunMetadataProvider';
+import {filterLogs} from './filterLogs';
 import {Container, DynamicRowContainer, Inner} from '../ui/VirtualizedTable';
 
 const BOTTOM_SCROLL_THRESHOLD_PX = 60;
 
-export const LogsScrollingTableNew = (props: ILogsScrollingTableProps) => {
+interface Props {
```
Comment (hellendag) [inline js_modules/dagster-ui/packages/ui-core/src/runs/LogsScrollingTableNew.tsx:15]: This is moved from the now-deleted `LogsScrollingTable`.

File: js_modules/dagster-ui/packages/ui-core/src/runs/LogsScrollingTableMessageFragment.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import {gql} from '@apollo/client';
+
+import {LOGS_ROW_STRUCTURED_FRAGMENT, LOGS_ROW_UNSTRUCTURED_FRAGMENT} from './LogsRow';
+
+export const LOGS_SCROLLING_TABLE_MESSAGE_FRAGMENT = gql`
```
Comment (hellendag) [inline js_modules/dagster-ui/packages/ui-core/src/runs/LogsScrollingTableMessageFragment.tsx:5]: This is moved from the now-deleted `LogsScrollingTable`.

**Pull request merged**